aten/src/ATen/cuda/nvrtc_stub/ATenNVRTC.h

```diff
@@ -64,6 +64,14 @@ namespace at { namespace cuda {
 //
 // HIP doesn't have
 //   cuGetErrorString  (maps to non-functional hipGetErrorString___)
+//
+// HIP from ROCm 3.5 on renamed hipOccupancyMaxActiveBlocksPerMultiprocessor
+// to hipModuleOccupancyMaxActiveBlocksPerMultiprocessor.
+#if HIP_VERSION < 305
+#define HIP_RENAME hipOccupancyMaxActiveBlocksPerMultiprocessor
```

Comment:
I think this macro name is bad. This is a header which means everyone will see this signature, but you've named this macro in a very local naming fashion.

Reply:
Chose a more self-documenting name.  Added the #undef to avoid polluting since the rename is local to the header.